data/whitelist.d/misc_whitelist.txt

```diff
@@ -102,3 +102,6 @@ www.sfr.fr
 www.tripadvisor.fr
 www.uptodown.com
 www.winamax.fr
+ps.pndsn.com
+api.mixpanel.com
```

Comment:
`api.mixpanel.com` is a tracking and analytics site. I think we should not whitelist this one.